.travis.yml

```diff
@@ -74,3 +74,13 @@ jobs:
 
 cache:
   yarn: true
+
+notifications:
+  webhooks:
+    urls:
+      - $TRAVIS_WEBHOOK
```

Comment:
Um. What is this.

Reply:
I'd guess a variable called `TRAVIS_WEBHOOK`.

Probably defined in the Travis CI web configuration.

Reply:
This is most definitely problematic. Is there a way we can make a variable from the URL without revealing it?

> Note: These webhooks are executed at the end of a build, and not by individual jobs (see builds vs jobs). This means that environment variables from the build are not available in this section.

Reply:
Why don't we reveal it? We can destroy and recreate the webhook in Discord very easily